src/taipy/core/_entity/_migrate.py

```diff
@@ -443,7 +451,7 @@ def __insert_version(version: dict, conn):
     conn.commit()
 
 
-def __write_entities_to_sql(_entities: Dict, _versions: Dict, db_file: str = "test.db"):
+def __write_entities_to_sql(_entities: Dict, _versions: Dict, db_file: str):
```

Comment:
We should have as a default value the default value of the configuration.
What do you think?